.github/actions/repo-gardening/src/tasks/notify-design/index.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+const { getInput } = require( '@actions/core' );
+
+/**
+ * Internal dependencies
+ */
+const debug = require( '../../debug' );
+const getLabels = require( '../../get-labels' );
+const sendSlackMessage = require( '../../send-slack-message' );
+
+/* global GitHub, WebhookPayloadPullRequest */
+
+/**
+ * Check for a Design Review status label on a PR.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasDesignReviewLabel( octokit, owner, repo, number ) {
+	const labels = await getLabels( octokit, owner, repo, number );
+	// We're only interested in the Needs Design Review label.
+	return !! labels.find( label => label.match( /^\[Status\]\sNeeds\sDesign\sReview$/ ) );
+}
+
+/**
+ * Check for a Needs Design label on a PR.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasNeedsDesignLabel( octokit, owner, repo, number ) {
+	const labels = await getLabels( octokit, owner, repo, number );
+	// We're only interested in the Needs Design label.
+	return !! labels.find( label => label.match( /^\[Status\]\sNeeds\sDesign$/ ) );
+}
+
+/**
+ * Check for a Design Input Requested label on a PR.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasInputRequestedLabel( octokit, owner, repo, number ) {
+	const labels = await getLabels( octokit, owner, repo, number );
+	// We're only interested in the Design Input Requested label.
+	return !! labels.find( label => label.match( /^\[Status\]\sDesign\sInput\sRequested$/ ) );
+}
+
+/**
+ * Send a Slack notification about a label to the Design team.
+ *
+ * @param {WebhookPayloadPullRequest} payload - Pull request event payload.
+ * @param {GitHub}                    octokit - Initialized Octokit REST client.
+ */
+async function notifyDesign( payload, octokit ) {
+	const { number, repository } = payload;
+	const { owner, name: repo } = repository;
+	const ownerLogin = owner.login;
+
+	const slackToken = getInput( 'slack_token' );
+	if ( ! slackToken ) {
+		debug( `notify-design: Input slack_token is required but missing. Aborting.` );
```

Comment:
Personally, I'd throw here and on line 82. Make the job fail if the required inputs are missing.

Reply:
Yeah, you're right, that's a good idea. Done in ab31b0bc99aec0796d9029e7dc532e5a16a41505